src/pages/Explorer/Category/Widget.svelte

```diff
@@ -6,14 +6,18 @@
   export let icon = 'info'
   export let color = 'green'
   export let iconWidth = 16
-  export let getItems = (page) => new Promise((resolve) => resolve([]))
+  export let getItems = (page) => new Promise((resolve) => resolve({ pages: 1, items: [] }))
```

Comment:
@peratik what is the reason for this default? I think should fail early during development if such import prop is missing